### PR TITLE
Initial commit for Scribe deployment via policy

### PR DESCRIPTION
**Pull request by cooktheryan**
Signed-off-by: Ryan Cook <rcook@redhat.com>

This is the initial commit for Scribe policy for RHACM. Description, specific directory placement, and any policies should be defined BEFORE merge.

### CODE REVIEW

**Comment by cooktheryan** [comment]
@ch-stark @berenss @jeff-brent

Ping

I think we will need to figure out specifically the directory this should go in and have a set of eyes whether this makes sense to land the way that it is defined.

**Comment by cooktheryan** [comment]
policy.open-cluster-management.io/standards: NIST 800-53
    policy.open-cluster-management.io/categories: CM Configuration Management
    policy.open-cluster-management.io/controls: CM-2 Baseline 
    
 do not specify a namespace
 
 
 Update readme

**Comment by cooktheryan** [comment]
@ycao56 feel free to ping me in Slack on any questions this is my first attempt at policy creation for the Scribe Epic.

File: community/CM-Configuration-Management/policy-persistent-data-management.yaml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                      kind: PlacementRule
+          remediationAction: enforce
+          severity: low
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: scribe-replication-placement
+        spec:
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-persistent-data-management.yaml]: `namespaceSelector` can be omitted as the namespace has been specified in object-templates

File: community/CM-Configuration-Management/policy-persistent-data-management.yaml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                  type: HelmRepo
+          remediationAction: enforce
+          severity: low
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-scribe-subscription
+        spec:
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-persistent-data-management.yaml]: `namespaceSelector` can be omitted as the namespace has been specified in object-templates

File: community/CM-Configuration-Management/policy-persistent-data-management.yaml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                        operator: In
+                        values:
+                          - scribe
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: scribe-replication-channel
+        spec:
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-persistent-data-management.yaml]: `namespaceSelector` can be omitted as the namespace has been specified in object-templates

File: community/CM-Configuration-Management/policy-persistent-data-management.yaml [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+          severity: high
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-scribe-application
+        spec:
+          remediationAction: enforce
+          severity: low
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-persistent-data-management.yaml]: `namespaceSelector` can be omitted as the namespace has been specified in object-templates

File: community/CM-Configuration-Management/policy-persistent-data-management.yaml [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+kind: PlacementRule
+metadata:
+  name: placement-scribe-policy-app
+spec:
+  clusterConditions:
+    - status: 'True'
+      type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchLabels:
+      name: local-cluster
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-persistent-data-management.yaml:149]: This policy is going to be applied on hub. Is this expected?

Comment (cooktheryan) [reply community/CM-Configuration-Management/policy-persistent-data-management.yaml:149]: @ycao56 the idea would be that the application would be defined on the hub the from there the placement on 139-140 would take over defining the placement on remote clusters

**Comment by ycao56** [comment]
Overall this pr looks good. However, I don't have the context of this work so can't really comment from the content perspective.

**Comment by cooktheryan** [comment]
@ycao56 I appreciate the review. From a goal perspective I had a call with @ch-stark this morning who seemed to be ok with the direction. He said you would be a great person to just make sure the overall format is correct and it matches the guidelines for RHACM

**Comment by ycao56** [comment]
> @ycao56 I appreciate the review. From a goal perspective I had a call with @ch-stark this morning who seemed to be ok with the direction. He said you would be a great person to just make sure the overall format is correct and it matches the guidelines for RHACM

Sounds good! Thanks for the PR.

**Review by ycao56 — APPROVED** [review]

**Pull request merged**